README.md

```diff
@@ -38,7 +38,27 @@ lineage:
     remote-url: https://github.com/felddy/extra-skel-sauce.git
 ```
 
-## Sample GitHub Actions workflow ##
+## Usage ##
+
+### Inputs ###
+
+| Name | Description | Default | Required |
+| ---- | ----------- | ------- | :------: |
+| access_token | GitHub personal access token. | n/a | yes |
+| mask_non_public_repos | Control the visibility of private repo names in the logs. | `true` | no |
+| public_repos_only | Control non-public repository processing. | `true` | no |
+| repo_query | GitHub search query to find repositories for pull requests. | n/a | yes |
```

Comment:
We should include a `Type` column in this table.  It may also be helpful to add example values for the required inputs (in the description field).

Reply:
Good thinking- I like `Interpreted Type` for this.

Reply:
Those changes looks great to me - thanks!